### PR TITLE
Add new task template to polyfills repo

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-task-template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: New Task Template
+about: Use this template when you are creating a new task
+title: ''
+labels: 'Type: Feature'
```
Comment (bicknellr) [inline .github/ISSUE_TEMPLATE/new-task-template.md:5]: Do tasks always imply features?

File: .github/ISSUE_TEMPLATE/new-task-template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: New Task Template
```
Comment (bicknellr) [inline .github/ISSUE_TEMPLATE/new-task-template.md:2]: What about just "Task"?

File: .github/ISSUE_TEMPLATE/new-task-template.md (hunk 1 above)
Comment (aomarks) [reply .github/ISSUE_TEMPLATE/new-task-template.md]: Yeah, doesn't quite make sense. I replaced this with 3 templates instead: Feature, Bug, Maintenance

File: .github/ISSUE_TEMPLATE/new-task-template.md (hunk 2 above)
Comment (aomarks) [reply .github/ISSUE_TEMPLATE/new-task-template.md]: (See above, replaced with 3 templates)

**Comment by aomarks** [comment]
@bicknellr PTAL, added 3 new templates to replace the one.

**Pull request merged**